example/src/screens/MeetingScreen/Modals.tsx

```diff
@@ -182,40 +171,36 @@ export const ParticipantsModal = ({
     if (filter === 'everyone') {
       setFilteredPeerTrackNodes(newFilteredPeerTrackNodes);
     } else if (filter === 'raised hand') {
-      const updatedPeerTrackNodes = newFilteredPeerTrackNodes?.filter(
-        peerTrackNode => {
-          const parsedMetaData = parseMetadata(peerTrackNode.peer?.metadata);
-          return parsedMetaData.isHandRaised === true;
-        },
-      );
+      const updatedPeerTrackNodes = newFilteredPeerTrackNodes?.filter(peer => {
+        const parsedMetaData = parseMetadata(peer?.metadata);
+        return parsedMetaData.isHandRaised === true;
+      });
       setFilteredPeerTrackNodes(updatedPeerTrackNodes);
     } else {
-      const updatedPeerTrackNodes = newFilteredPeerTrackNodes?.filter(
-        peerTrackNode => {
-          return peerTrackNode.peer?.role?.name === filter;
-        },
-      );
+      const updatedPeerTrackNodes = newFilteredPeerTrackNodes?.filter(peer => {
+        return peer?.role?.name === filter;
+      });
       setFilteredPeerTrackNodes(updatedPeerTrackNodes);
     }
-  }, [participantsSearchInput, filter, peerTrackNodes]);
+  }, [participantsSearchInput, filter, hmsPeers]);
 
   useEffect(() => {
-    let newPeerCount = 0;
-    peerTrackNodes?.map(peerTrackNode => {
-      if (peerTrackNode.id.includes(HMSTrackSource.REGULAR)) {
-        newPeerCount++;
+    instance?.getRemotePeers().then(peers => {
```

Comment:
`getRemotePeers` function name is misleading when actually it returns all - local + remote peers

Reply:
local peer is taken from local state not from the api. its only returning remote peers